benchmarking/commons/hpo_main_local.py

```diff
@@ -172,25 +169,26 @@ def create_objects_for_tuner(
         metadata=metadata,
         save_tuner=configuration.save_tuner,
     )
+    callbacks = [StoreResultsCallback(final_results_composer=final_results)]
     if configuration.remote_tuning_metrics:
         # Use callback to report tuning metrics
-        tuner_kwargs["callbacks"] = [
-            StoreResultsCallback(),
+        callbacks.append(
             RemoteTuningMetricsCallback(
                 metric=benchmark.metric,
                 mode=benchmark.mode,
                 config_space=benchmark.config_space,
                 resource_attr=benchmark.resource_attr,
-            ),
-        ]
+            )
+        )
+    tuner_kwargs["callbacks"] = callbacks
     return tuner_kwargs
 
 
 def start_benchmark_local_backend(
     configuration: ConfigDict,
     methods: MethodDefinitions,
     benchmark_definitions: RealBenchmarkDefinitions,
-    post_processing: Optional[PostProcessingType] = None,
+    final_results: Optional[FinalResultsComposer] = None,
```

Comment:
Replaces ``post_processing``. Instead of running some code at the end, we extract some extra results which are stored. This is more useful.

Reply:
Are there any known uses of customers using `post_processing`? 

I agree that `final_results` is a great new feature -- just wondering before we remove `post_processing` if there are any use-cases that will break from the removal.

Especially since this is in `benchmarking`, I think it's very low-risk to update.

Reply:
Good point. I don't think anybody except me uses this right now.